receiver/simpleprometheusreceiver/receiver.go

```diff
@@ -78,13 +95,17 @@ func getPrometheusConfig(cfg *Config) (*prometheusreceiver.Config, error) {
 
 	scheme := "http"
 
-	if cfg.TLSEnabled {
+	tlsConfig, err := cfg.TLSSetting.LoadTLSConfig()
```

Comment:
Shouldn't `cfg.HTTPClientSettings.TLSSetting.Insecure` be set to `true` by default to get `tlsConfig = nil`?

Reply:
> Did you test the receiver? I doesn't look like working for regular http endpoints

Sorry, I didn't test it sufficiently, so this time I add another test Function `TestGetPrometheusConfigWrapper` to test the new function I add getPrometheusConfigWrapper, in 4 cases

- `tls_enable` configured true, `tls` default config,  which results to https
- `tls_enable` configured false(default), `tls` default config,  which results to http
- `tls_enable` configured false(default), `tls` config to use client security,  which results to https
- `tls_enable` configured false(default), `tls` config to use client security, and specify ca path,  which results to https